setup.py

```diff
@@ -390,6 +390,7 @@ def is_Numpy_installed():
     'Bio.Affy',
     'Bio.Cluster',
     'Bio.KDTree',
+    'Bio.phenotype'
```

Comment:
Ideally that would have included a trailing comma, so that next time an entry is added to the list it is a one line change. But there's no real need to change it now.